### PR TITLE
Add previews on versions tab

### LINKED ISSUE #1670: Versionen-Reiter mit Vorschaubilder erweitern
Der Reiter "Versionen" soll mit der Vorschaufunktion erweitert werden.

bildschirmfoto 2016-04-05 um 14 25 58

### PR DESCRIPTION
**Pull request by elioschmutz**
This PR add the bumblebee-previews for the document-versions:

bildschirmfoto 2016-06-24 um 16 50 53

closes #1670

### CODE REVIEW

File: opengever/bumblebee/browser/templates/bumblebeeoverlay.pt [hunk 1]
```diff
@@ -69,10 +72,23 @@
...
+        </li>
       </ul>
       <div class="info-viewlets">
         <div tal:replace="structure view/overlay/render_checked_out_viewlet" />
         <div tal:replace="structure view/overlay/render_lock_info_viewlet" />
+        <div tal:condition="view/overlay/is_versioned_context">
+          <dl class="portalMessage warning">
+            <dt i18n:domain="plone" i18n:translate="">Warning</dt>
+            <dd i18n:translate="warning_on_versioned_context">
+              You are watching a versioned file.
```
Comment (lukasgraf) [inline opengever/bumblebee/browser/templates/bumblebeeoverlay.pt]: `watching` -> `looking at`

File: opengever/bumblebee/locales/de/LC_MESSAGES/opengever.bumblebee.po [hunk 2]
```diff
@@ -14,41 +14,51 @@ msgstr ""
...
 
-#: ./opengever/bumblebee/browser/overlay_templates/bumblebeeoverlayview.pt:25
+#: ./opengever/bumblebee/browser/templates/bumblebeeoverlay.pt:25
 msgid "Sequence number:"
 msgstr "Laufnummer:"
 
+#. Default: "Revert document"
+#: ./opengever/bumblebee/browser/overlay.py:190
+msgid "label_revert"
+msgstr "Dokument zurücksetzten"
```
Comment (deiferni) [inline opengever/bumblebee/locales/de/LC_MESSAGES/opengever.bumblebee.po]: typo

File: opengever/document/locales/de/LC_MESSAGES/opengever.document.po [hunk 3]
```diff
@@ -432,10 +433,15 @@ msgid "label_related_documents"
...
 #. Default: "reset"
-#: ./opengever/document/browser/versions_tab.py:162
+#: ./opengever/document/browser/versions_tab.py:163
 msgid "label_reset"
 msgstr "Zurücksetzen"
 
+#. Default: "Revert"
+#: ./opengever/document/browser/versions_tab.py:324
+msgid "label_revert"
+msgstr "Zurücksetzten"
```
Comment (deiferni) [inline opengever/document/locales/de/LC_MESSAGES/opengever.document.po]: typo

File: docs/HISTORY.txt [hunk 4]
```diff
@@ -5,6 +5,9 @@ Changelog
 4.9.0 (unreleased)
 ------------------
 
+- Implement bumblebee preview images in versions-tab.
```
Comment (deiferni) [inline docs/HISTORY.txt]: preview images in -> overlay for the

**Comment by elioschmutz** [comment]
@deiferni ready t merge

**Comment by deiferni** [comment]
👍

**Pull request merged**